modules/service/src/main/java/com/webank/wedatasphere/exchangis/http/converters/CustomHttpMessageConverter.java

```diff
@@ -60,6 +66,26 @@ public void addSupportedMediaTypes(){
             mediaTypes.addAll(MediaType.parseMediaTypes(supportMedias));
         }
         setSupportedMediaTypes(mediaTypes);
+
+        ObjectMapper mapper = this.getObjectMapper();
+        // jackson date config
+        if (this.jacksonProperties.getTimeZone() != null) {
+            mapper.setTimeZone(this.jacksonProperties.getTimeZone());
+        }
+
+        if (this.jacksonProperties.getDateFormat() != null) {
+            String dateFormat = jacksonProperties.getDateFormat();
+            SimpleDateFormat simpleDateFormat = new SimpleDateFormat(
+                dateFormat);
+
+            TimeZone timeZone = this.jacksonProperties.getTimeZone();
+            if (timeZone == null) {
+                timeZone = new ObjectMapper().getSerializationConfig()
```

Comment:
Don't new ObjectMaper() again

Reply:
You can consult the class "org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration"